apps/src/utils.js

```diff
@@ -141,6 +141,17 @@ exports.wrapNumberValidatorsForLevelBuilder = function () {
 };
 
 /**
+ * Return a random key name from an object.
+ *
+ * Slightly modified from: http://stackoverflow.com/a/15106541
+ */
+
+exports.randomKey = function (obj) {
+  var keys = Object.keys(obj);
+  return keys[keys.length * Math.random() << 0];
```

Comment:
Yes, for positive numbers that can be expressed as 32-bit integers. It truncates, so it behaves differently for negative numbers. I spent some time researching this after copying this code.

Apparently, it's also faster than Math.floor()